README.md

```diff
@@ -60,6 +60,27 @@ During the tutorial, don't hesitate to ask for help:
   can be posted.
 
 
+## Outline of the tutorial
+The tutorial will take you through stages to go a from a script (stage 0) to 
+the packaged application. Exercises will get you to build the most significant 
+portions of the differences between each stages but each of them is provided as 
+a separate folder are provided to make sure everyone to get to the end point:
```

Comment:
Maybe this as an edit?
"The tutorial will guide you through all the stages from a basic python script (stage 0) to a fully packaged and installable application. The included exercises will walk you through developing the primary product of each stage. A solution is available for each exercise, though, to ensure all participants are able to reach the end goal."

Reply:
Thanks. Done.